torch/distributed/_shard/sharding_spec/chunk_sharding_spec_ops/_common.py

```diff
@@ -57,7 +57,7 @@ def _register_sharded_op_on_local_tensor(
     """
     @sharded_op_impl(op)
     @_sharded_op_common(op, early_stop_func, extra_check)
-    def sharded_tensor_op_on_local_shards(types, args=(), kwargs=None, pg=None):
+    def sharded_tensor_op_on_local_tensor(types, args=(), kwargs=None, pg=None):
```

Comment:
hmmm we didn't change the callsites of `_register_sharded_op_on_local_shards` to `_register_sharded_op_on_local_tensor`, is this already been changes somewhere else?

Reply:
Nope, we don't need to change the callsite because this is the func inside the decorator. And we are not changing the signature of decorator itself.